dist/constants.json

```diff
@@ -6923,899 +6923,899 @@
       },
       {
         "value": "671GK",
-        "label": "SAN ANTONIO VA CLINIC"
+        "label": "San Antonio VA Clinic"
       },
       {
         "value": "671GL",
-        "label": "NEW BRAUNFELS VA CLINIC"
+        "label": "New Braunfels VA Clinic"
       },
       {
         "value": "671GN",
-        "label": "SEGUIN VA CLINIC"
+        "label": "Seguin VA Clinic"
       },
       {
         "value": "549GJ",
-        "label": "SHERMAN VA CLINIC"
+        "label": "Sherman VA Clinic"
       },
       {
         "value": "674HB",
-        "label": "LAGRANGE VA CLINIC"
+        "label": "LaGrange VA Clinic"
       },
       {
         "value": "740GA",
-        "label": "HARLINGEN OPC"
+        "label": "Harlingen VA Clinic-Treasure Hills"
       },
       {
         "value": "740GC",
-        "label": "CORPUS CHRISTI OPC"
+        "label": "Corpus Christi VA Clinic"
       },
       {
         "value": "740GD",
-        "label": "LAREDO OPC"
+        "label": "Laredo VA Clinic"
       },
       {
         "value": "740GB",
-        "label": "MCALLEN OPC"
+        "label": "McAllen VA Clinic"
       },
       {
         "value": "740GE",
         "label": "EAGLE PASS CBOC"
       },
       {
         "value": "756GB",
-        "label": "EL PASO EASTSIDE CBOC"
+        "label": "El Paso Eastside VA Clinic"
       },
       {
         "value": "580GF",
-        "label": "LAKE JACKSON CBOC"
+        "label": "Lake Jackson VA Clinic"
       },
       {
         "value": "580GG",
-        "label": "RICHMOND CBOC"
+        "label": "Richmond VA Clinic"
       },
       {
         "value": "674A4",
-        "label": "WACO, TX VAMC"
+        "label": "Doris Miller Department of Veterans Affairs Medical Center"
```

Comment:
- Consider abbreviated label for "Department of Veterans Affairs"
- Let's confirm this name change.